src/app/doctor/[id]/components/activity/activity.module.css

```
@@ -0,0 +1,19 @@
+.activity {
+  .card {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+
+    .activity-item {
+      display: flex;
+      align-items: center;
+      gap: 0.5rem;
+
+      background-color: var(--color-primary-opposite);
+
+      > svg {
+        font-size: var(--fz-500);
```

این ممکنه تغییراتش شدیدتر از متن کناری باشه
بهتره بر اساس em بذارید که همیشه متناسب با فونت متن کناریش تغییر کنه

```suggestion
        font-size: 1.25em;
```